### PR TITLE
fix: Table tooltip appears centered above header text

### PR DESCRIPTION
**Pull request by kaydbwithers**
## Why
This PR fixes Table styling to allow the tooltip to appear correctly above Table header text.

## What
Before
image
image

After
image
image

### CODE REVIEW

File: draft-packages/table/KaizenDraft/Table/Table.module.scss [hunk 1]
```diff
@@ -72,8 +68,6 @@ $row-height-data-variant: 48px;
 // The .headerRowCell wrapper is required to increase the specificity, as so
 // we can override the `Tooltip` component's styling.
 .headerRowCell .headerRowCellTooltip {
```
Comment (gyfchong) [inline draft-packages/table/KaizenDraft/Table/Table.module.scss:70]: Have you had a look at changing the [Tooltip display prop](https://cultureamp.design/storybook/?path=/docs/components-tooltip--docs)? Might remove the need to apply any extra styling.

Comment (kaydbwithers) [reply draft-packages/table/KaizenDraft/Table/Table.module.scss:70]: Thanks @gyfchong, I tried but it doesn't seem to help. The problem is with the header row expanding the full width.

**Review by gyfchong — APPROVED** [review]

File: .changeset/smooth-dogs-cheat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-table": minor
```
Comment (gyfchong) [inline .changeset/smooth-dogs-cheat.md:2]: I'd say this is a `patch` since no new feature was added.

Comment (kaydbwithers) [reply .changeset/smooth-dogs-cheat.md]: Thanks @gyfchong, that's done.

**Review by gyfchong — APPROVED** [review]

**Review by gyfchong — APPROVED** [review]

**Pull request merged**